source/archive/rfc/111.pod

```diff
@@ -24,7 +24,7 @@ this up.
 
 =head1 DESCRIPTION
 
-Perl6 should ignore any whitespace before the terminator of a heredoc on any
+Raku should ignore any whitespace before the terminator of a heredoc on any
```

Comment:
Definitely, not this one. These are RFCs, and should stay just the way they were created by their authors.